packages/components/tests/dummy/app/templates/components/stepper.hbs

```diff
@@ -106,7 +102,9 @@
     '
   />
   <p class="dummy-paragraph">Renders to:</p>
-  <Hds::Stepper::Indicator::Step @text="1" @status="incomplete" />
+  <Hds::Stepper>
+    <Hds::Stepper::Indicator::Step @text="1" @status="incomplete" />
```

Comment:
[non-blocking]: do we have guidelines on when to use a contextual component and when to compose namespace components? I go back and forth tbh, but I do like the composition here.